pvlib/atmosphere.py

```diff
@@ -158,9 +158,11 @@ def get_relative_airmass(zenith, model='kastenyoung1989'):
         * 'gueymard1993' - See reference [4] -
           requires apparent sun zenith
         * 'young1994' - See reference [5] -
-          requries true sun zenith
+          requires true sun zenith
         * 'pickering2002' - See reference [6] -
           requires apparent sun zenith
+        * 'gueymard2003' - See reference [7] -
+          requires true sun zenith
```

Comment:
I've put "true sun zenith" here because I found no mention of refraction correction, "apparent" etc in the reference, but it would be good if a reviewer could determine this with more confidence.

Reply:
I suspect it should be "true", and I suspect it should be "true" for Gueymard1993 as well.  I've sent him a note to ask.

Reply:
Chris also commented that the default should be a more modern model.

Reply:
> Chris also commented that the default should be a more modern model.

Does he have specific recommendations?

Reply:
Implicitly, Gueymard 2003.

Reply:
Thanks @adriesse for checking!

Updated default: no objection from me, but I think it will have to wait for 0.10.0.

Reply:
In a separate issue.  I think there could be some general discussion about defaults actually.